controller/api/proxy/endpoint_listener_test.go

```diff
@@ -223,11 +230,12 @@ func TestEndpointListener(t *testing.T) {
 		}
 
 		mockGetServer := &mockDestinationGetServer{updatesReceived: []*pb.Update{}}
-		listener := &endpointListener{
-			ownerKindAndName: ownerKindAndName,
-			stream:           mockGetServer,
-			enableTLS:        true,
-		}
+		listener := newEndpointListener(
+			mockGetServer,
+			ownerKindAndName,
+			true,
```

Comment:
I find explicit field names to be very helpful for boolean fields like this.  mind bringing them back?

Reply:
Agree i prefer explicit field names. In this case, we're replacing:
```go
listener := &endpointListener{...}
```
with:
```go
listener := newEndpointListener(...)
```
This change was motivated by the instantiation of `endpointListener.log` in `newEndpointListener()`. We could alternatively instantiate the log here via `&endpointListener{log: ...}`, but calling `newEndpointListener()` gives slightly more test coverage.